src/components/Image.js

```diff
@@ -73,7 +73,9 @@ export const Image = ({
         PlaceholderContent={PlaceholderContent}
         placeholderStyle={{ backgroundColor: colors.transparent }}
         accessible={!!source?.captionText}
-        accessibilityLabel={`(${source?.captionText}) ${consts.a11yLabel.image}`}
+        accessibilityLabel={`(${source.captionText ? source.captionText : ''}) ${
```

Comment:
this way there will be empty parentheses, if there is no `captionText`.
the image text is already in parentheses as well, so are they needed here at all?

Reply:
I think we can do without parentheses around  `captionText` if the empty parentheses can cause a sort of issue, is this the case ? 
I'm using parentheses around every text, perhaps an excessive precaution?

Reply:
The parentheses are not really necessary her, i agree.